erpnext/manufacturing/doctype/work_order/work_order.js

```diff
@@ -119,6 +119,10 @@ frappe.ui.form.on("Work Order", {
 			});
 			erpnext.work_order.set_default_warehouse(frm);
 		}
+
+		if(frm.fields_dict["operations"].get_value().length == 0){
+			frm.fields_dict["operations_section"].collapse();
+		}
```

Comment:
Use collapsible property on doctype instead? This completely takes away the ability to add new rows.

<img width="611" alt="Screenshot 2021-11-03 at 1 04 05 PM" src="https://user-images.githubusercontent.com/9079960/140023259-6051714b-b79d-4a87-a5d4-c60ee0832cae.png">

Reply:
I had enabled the collapsible property since this issue was happening locally and then it was fixed need to check what is the problem now. was collapsible property not enabled when you were checking ?